src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-conditions.mdx

```diff
@@ -186,6 +188,9 @@ Creating a buffer:
   Key considerations:
    * <DoNotTranslate>**Popular use case**</DoNotTranslate>: A common use of gap filling is to insert a value of 0 for windows with no data received.
    * <DoNotTranslate>**Anomaly thresholds**</DoNotTranslate>: The gap-filling value is interpreted as the number of standard deviations from the last observed value when using anomaly thresholds. For example, filling gaps with a value of 0 would replicate the last value seen, effectively assuming no change.
+  * <DoNotTranslate>**Cost**</DoNotTranslate>: In an alert condition you can use a `WHERE` clause to filter for the specific transactions in an application you want to count. For example, if you're monitoring your `WebPortal` application and there are no transactions during a designated time period; the alert condition will not register any data, the entity status will change from green to gray, and, in some cases, incidents won't be created or closed. Suppose this gap is less than two hours. In that case, it can be addressed using the gap-filling strategy outlined in our [Create NRQL alert conditions documentation](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#example-null-zero). For cases where the data gaps exceed  2 hours or if a new data point is expected, we recommend using [signal loss threshold](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-conditions/#lost-signal). **Remember that this strategy can increase compute power usage, affecting your bill**.
```

Comment:
There's an odd semicolon after "during a designated time period;"

```suggestion
  * <DoNotTranslate>**Cost**</DoNotTranslate>: In an alert condition you can use a `WHERE` clause to filter for the specific transactions in an application you want to count. For example, if you're monitoring your `WebPortal` application and there are no transactions during a designated time period, the alert condition will not register any data, the entity status will change from green to gray, and, in some cases, incidents won't be created or closed. Suppose this gap is less than two hours. In that case, it can be addressed using the gap-filling strategy outlined in our [Create NRQL alert conditions documentation](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#example-null-zero). For cases where the data gaps exceed  2 hours or if a new data point is expected, we recommend using [signal loss threshold](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-conditions/#lost-signal). **Remember that this strategy can increase compute power usage, affecting your bill**.
```

Reply:
I don't think this quite makes sense:

"**Remember that this strategy can increase compute power usage, affecting your bill**."

I don't think alerts are currently billed this way...